lnwallet/btcwallet/signer_test.go

```diff
@@ -167,6 +173,25 @@ func TestBip32KeyDerivation(t *testing.T) {
 	}
 }
 
+func TestScriptImport(t *testing.T) {
+	netParams := &chaincfg.RegressionNetParams
+	w, cleanup := newTestWallet(t, netParams, seedBytes)
+	defer cleanup()
+
+	firstDerivedAddr, err := w.NewAddress(
+		lnwallet.TaprootPubkey, false, lnwallet.DefaultAccountName,
+	)
+	require.NoError(t, err)
+	require.Equal(t, firstAddressTaproot, firstDerivedAddr.String())
+
+	_, err = w.InternalWallet().Manager.FetchScopedKeyManager(
+		waddrmgr.KeyScopeBIP0086,
+	)
+	require.NoError(t, err)
+
+	// mgr.ImportWitnessScript()
```

Comment:
Leaving a comment here so this doesn't get lost.

Reply:
Thanks, added a TODO comment and split into its own commit.